### PR TITLE
Editorial updates to glossary terms

### PR DESCRIPTION
**Pull request by bobgt**
Editorial suggestions to glossary terms with abbreviations.

### CODE REVIEW

File: glossary.yaml [hunk 1]
```diff
@@ -1,10 +1,8 @@
 terms:
   - definition: "An account is a unique identifier and a requirement to interact with an EOSIO blockchain. Unlike most other cryptocurrencies, transfers are sent to a human readable account name instead of a public key, while keys attributed to the account are used to sign transactions."
     name: Account
-  - definition: "The Application Binary Interface (ABI) is a JSON-based description on how to convert user actions between their JSON and binary representations. The ABI may also describe how to convert the database state to/from JSON. Once you have described your contract via an ABI this allows developers and users to interact with your contract seamlessly via JSON."
-    name: ABI
-    synonyms:
-      - "Application Binary Interface"
+  - definition: "A JSON-based description on how to convert user actions between their JSON and binary representations. The ABI may also describe how to convert the database state to/from JSON. Once you have described your contract via an ABI this allows developers and users to interact with your contract seamlessly via JSON."
+    name: Application Binary Interface (ABI)
```
[the remark this answers is not in the document]
Comment (bobgt) [reply glossary.yaml]: Added `Abbreviation` meta-type.

File: glossary.yaml [hunk 2]
```diff
@@ -13,18 +11,12 @@ terms:
...
-  - definition: "DPoS stands for \"Delegated Proof of Stake\" and is a consensus algorithm initially developed by Daniel Larimer in 2013 for Bitshares. It's sometimes referred to as \"Democracy as Proof of Stake\""
-    name: DPoS
-    synonyms:
-      - "Delegated Proof-of-Stake"
+  - definition: "An entity that utilizes a combination of automation and input from stakeholders, normally in the form of votes. They are often governed by code which describes the purpose of the organisation."
+    name: Decentralized Autonomous Company (DAC)
+  - definition: "A consensus of replicated, shared, and synchronized digital data geographically spread across multiple sites, countries, or institutions. DLT is also called a shared ledger."
+    name: Distributed Ledger Technology (DLT)
+  - definition: "A consensus algorithm initially developed by Daniel Larimer in 2013 for Bitshares. It's sometimes referred to as \"Democracy as Proof of Stake\"".
+    name: Delegated Proof of Stake (DPoS)
```
[the remark this answers is not in the document]
Comment (bobgt) [reply glossary.yaml]: Added Abbreviation meta-type

File: glossary.yaml [hunk 3]
```diff
@@ -168,8 +160,8 @@ terms:
     name: "EOSIO Types"
   - definition: "In order to transfer transaction content between nodes faster and to save storage space when storing transaction content in and EOSIO based blockchain database, the transactions are 'translated' from json into a packed form which is smaller in size. To get the packed version of a transaction one can use the cleos convert command."
     name: "Packed Transaction"
-  - definition: "WIF stands for Wallet Import Format and is an encoding for a private EDSA key. EOSIO uses the same version, checksum, and encoding scheme as the Bitcoin WIF addresses and should be compatible with existing libraries. The following is an example of a WIF Private Key: 5HpHagT65TZzG1PH3CSu63k8DbpvD8s5ip4nEB3kEsreAbuatmU"
-    name: WIF
+  - definition: "An encoding for a private EDSA key. EOSIO uses the same version, checksum, and encoding scheme as the Bitcoin WIF addresses and should be compatible with existing libraries. The following is an example of a WIF Private Key: 5HpHagT65TZzG1PH3CSu63k8DbpvD8s5ip4nEB3kEsreAbuatmU"
+    name: Wallet Import Format (WIF)
```
[the remark this answers is not in the document]
Comment (bobgt) [reply glossary.yaml]: Added Abbreviation meta-type.

File: glossary.yaml [hunk 4]
```diff
@@ -180,8 +172,8 @@ terms:
...
     name: "Unconfirmed Transaction"
-  - definition: "WASM stands for WebAssembly. The EOSIO based blockchains execute user-generated applications and code using WebAssembly. WASM is an emerging web standard with widespread support of Google, Microsoft, Apple, and others. At the moment the most mature toolchain for building applications that compile to WASM is clang/llvm with their C/C++ compiler. For best compatibility, it is recommended that you use the EOSIO toolchain to generate WASM."
-    name: WASM
+  - definition: "The EOSIO based blockchains execute user-generated applications and code using WebAssembly. WASM is an emerging web standard with widespread support of Google, Microsoft, Apple, and others. At the moment the most mature toolchain for building applications that compile to WASM is clang/llvm with their C/C++ compiler. For best compatibility, it is recommended that you use the EOSIO toolchain to generate WASM."
+    name: WebAssembly (WASM)
```
[the remark this answers is not in the document]
Comment (bobgt) [reply glossary.yaml]: Added Abbreviation meta-type.

**Comment by dskvr** [comment]
@bobgt updated your branch, in general, parameters aren't `Uppercase` or `PascalCase`, and pluralized.

`Abbreviation -> abbreviations`

**Pull request merged**